rules/windows/credential_access_posh_request_ticket.toml

```diff
@@ -58,6 +58,24 @@ systems, and web services. Prioritize privileged accounts.
 - Determine the initial vector abused by the attacker and take action to prevent reinfection through the same vector.
 - Using the incident response data, update logging and audit policies to improve the mean time to detect (MTTD) and the
 mean time to respond (MTTR).
+
+## Config
+
+The 'PowerShell Script Block Logging' logging policy must be enabled.
+Steps to implement the logging policy with with Advanced Audit Configuration:
+
+```
+Computer Configuration > 
+Administrative Templates > 
+Windows PowerShell > 
+Turn on PowerShell Script Block Logging (Enable)
+```
+
+Steps to implement the logging policy via registry:
```

Comment:
```suggestion
Steps for implementing the logging policy through the registry:
```

Reply:
I'll leave this one out by now, and apply a fix accross all rules that use this verbiage in a separated PR